gnucash/register/register-gnome/gnucash-sheet-private.c

```diff
@@ -514,6 +514,24 @@ draw_cell (GnucashSheet *sheet, SheetBlock *block,
     }
 #endif
 
+    if ((text != NULL) && (*text != '\0'))
+    {
+        if (g_strcmp0 (PRICE_CELL_TYPE_NAME, gnc_table_get_cell_type_name (table, virt_loc)) == 0)
+        {
```

Comment:
Please combine the `if`s with `&&` and split the `g_strcmp0` args on the comma to reduce the indentation and line width.